com.unity.render-pipelines.universal/Editor/Shadow/ShadowCascadeSplitGUI.cs

```diff
@@ -56,7 +56,7 @@ public DragCache(int activePartition, float normalizedPartitionSize, Vector2 cur
             *  @param  normalizedCascadePartition      The array of partition sizes in the range 0.0f - 1.0f; expects ONE entry if cascades = 2, and THREE if cascades=4
             *                                          The last entry will be automatically determined by summing up the array, and doing 1.0f - sum
             */
-        public static void HandleCascadeSliderGUI(ref float[] normalizedCascadePartitions)
+        public static void HandleCascadeSliderGUI(ref float[] normalizedCascadePartitions, float distance, EditorUtils.Unit unit)
```

Comment:
public API change?

Reply:
Added original function to deprecated and made it obsolete